misc/services/mobile-webui/mobile-webui-frontend/src/containers/activities/picking/PickStepScreen.jsx

```diff
@@ -88,40 +89,30 @@ class PickStepScreen extends Component {
       qtyRejectedReasonCode: 'N',
       huBarcode,
     }).then(() => {
-      dispatch(
-        updatePickingStepQty({
-          wfProcessId,
-          activityId,
-          lineId,
-          stepId,
-          altStepId,
-          qtyPicked: 0,
-          qtyRejected,
-          qtyRejectedReasonCode: 'N',
-        })
-      );
+      updatePickingStepQty({
+        wfProcessId,
+        activityId,
+        lineId,
+        stepId,
+        altStepId,
+        qtyPicked: 0,
+        qtyRejected,
+        qtyRejectedReasonCode: 'N',
+      });
     });
   };
 
-  componentWillUnmount() {
-    const { wfProcessId, activityId, lineId, stepId, dispatch } = this.props;
+  onScanButtonClick = () => {
+    const { push } = this.props;
+    const location = pickingStepScanScreenLocation(this.props);
```

Comment:
send only the required params

Reply:
think that we are moving stocks of 100k EUR... it's very important to be very precise.